.github/workflows/chromatic.yaml

```diff
@@ -0,0 +1,79 @@
+# Chromatic action from https://www.chromatic.com/docs/github-actions
+
+#v2
+
+# Workflow name
+name: "chromatic"
+
+on: push
+
+# List of jobs
+jobs:
+  visual-testing:
+    # Operating System
+    runs-on: ubuntu-latest
+
+    # Job steps
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
+
+      - uses: 8BitJonny/gh-get-current-pr@1.1.0
+        id: getPr
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # Only return if PR is still open
+          filterOutClosed: true
+
+      - run: echo "The current PR number is ${prNumber}"
+        if: success() && steps.getPr.outputs.number
+        env:
+          prNumber: ${{ steps.getPr.outputs.number }}
+
+      - name: Get most recent comment
+        if: success() && steps.getPr.outputs.number
+        uses: peter-evans/find-comment@v1
+        id: latestComment
+        with:
+          issue-number: ${{ steps.getPr.outputs.number }}
+          body-includes: ""
+          direction: last
+
+      - run: |
+          echo ${{ steps.latestComment.outputs.comment-id }}
+          echo ${{ steps.latestComment.outputs.comment-body }}
+
+      - run: echo "found run chromatic text in latest comment"
+        if: contains(steps.latestComment.outputs.comment-body, 'run chromatic')
+
+      - uses: actions/cache@v2 # use of cache for node_modules saves usually > 1min of build time
+        if: contains(steps.latestComment.outputs.comment-body, 'run chromatic')
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-
+            ${{ runner.os }}-
+
+      - uses: actions/setup-node@v1
+        if: contains(steps.latestComment.outputs.comment-body, 'run chromatic')
```

Comment:
I'm sure there's a way to turn this into a variable to avoid repetition, but I don't know how!